website/docs/r/kubernetes_cluster.html.markdown

```diff
@@ -182,6 +182,8 @@ resource "azurerm_kubernetes_cluster" "example" {
 
 * `windows_profile` - (Optional) A `windows_profile` block as defined below.
 
+* `http_proxy_config` - (Optional) A `http_proxy_config` block as defined below.
```

Comment:
can we sort this alphabetically in the list of optional fields (e.g. move this up)?

in addition can we also add a note that this functionality is in Preview here:

```suggestion
* `http_proxy_config` - (Optional) A `http_proxy_config` block as defined below.

-> **Note:** HTTP/HTTPS Proxy support is in Public Preview.
```